### PR TITLE
add missing plugin install+update doit subtask

### PR DESCRIPTION
**Pull request by Ebaneck**
**Component**: Documentation

**Context**: Updating documentation from Local Dev Standpoint

**Summary**:
Proposes a few new changes to how the Readme documentation is outlined while including a missing & mandatory bootstrapping step.

- The missing step happens to be the VirtualBox guest addition which is required for bootstrapping a local dev environment.

Also, a few changes have been made to the Build doc relating to `doit` sub-commands ie vagrant_up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,20 +15,27 @@ For more information, please refers to [BUILDING.md](BUILDING.md).
...
+vagrant plugin install vagrant-vbguest
 # Bootstrap a platform on a vagrant environment using
-./doit.sh vagrantup
+./doit.sh vagrant_up
+```
+
+### End-to-End Testing
+
+To run tests locally: Complete the bootstrapping step above
+```shell
```
Comment (gdemonet) [inline README.md:38]: While you're at it, we don't need to generate the SSH config file by hand anymore, it's handled in `tox`. You could update this code block with something like:

```shell
# Run tests with tox
tox -e tests
```

And optionally (but I think we could do it in another PR, when we extract the Terraform tooling), mention how one can define his own `SSH_CONFIG_FILE` and `SSH_HOSTS_LIST` environment vars to affect the `tox -e tests` invocation.

**Review by gdemonet — DISMISSED** [review]
LGTM, just a note on running E2E tests which is not up-to-date either

File: README.md [hunk 2]
```diff
@@ -15,20 +15,27 @@ For more information, please refers to [BUILDING.md](BUILDING.md).
...
+## Testing
 ### Requirements
 
 - [Python3.6+](https://www.python.org/)
 - [tox](https://pypi.org/project/tox)
 - [Vagrant](https://www.vagrantup.com/)
 - [VirtualBox](https://www.virtualbox.org)
 
-### Usage
+### Bootstrapping local environment
```
Comment (NicolasT) [inline README.md]: `Bootstrapping a local environment`

File: README.md [hunk 3]
```diff
@@ -15,20 +15,27 @@ For more information, please refers to [BUILDING.md](BUILDING.md).
...
+# Install virtualbox guest addition plugin
+vagrant plugin install vagrant-vbguest
 # Bootstrap a platform on a vagrant environment using
-./doit.sh vagrantup
+./doit.sh vagrant_up
+```
+
+### End-to-End Testing
+
+To run tests locally: Complete the bootstrapping step above
```
Comment (NicolasT) [inline README.md]: `To run the test-suite locally, first complete the bootstrap step as outline above.`

File: README.md (hunk 2 above)
Comment (Ebaneck) [reply README.md]: Thanks.. this is resolved

File: README.md (hunk 1 above)
Comment (Ebaneck) [reply README.md:38]: Thanks for the heads-up @gdemonet

File: README.md (hunk 2 above)
Comment (NicolasT) [reply README.md]: But not pushed?

**Review by NicolasT — APPROVED** [review]

**Comment by Ebaneck** [comment]
@bert-e approve
